### DISCUSSION TITLE
Is it possible to query the API with non-english pokemon names?

### DISCUSSION THREAD

**Discussion by AFoeee** [PokeAPI/pokeapi#701] [Q&A] [answered]
Is there a way to search for a pokemon via one of its non-english names?

For example, what would a query look like if I searched for `Glumanda` (the german name of `Charmander`)?

**Comment by Naramsim** [comment, marked as answer]
No, that's not possible at the moment. Using GraphQL would be possible. https://beta.pokeapi.co/graphql/console/
